INSTALL

```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
```

Comment:
Thanks for pointing out the inclusion in the built docs - that's very helpful.

However, I still think we should avoid this markup:
 1. When used to create the built docs http[s] links are *already* converted to clickable links. There is no need to invoke markup. This is the style used in the vast majority of the rest of the file. You already have two other clickable links so perhaps the simplest change is just to remove this link.
 2. When viewed as a text file the markup just makes it harder to read.